waspc/src/Wasp/Generator/EmailSenders.hs

```diff
@@ -0,0 +1,23 @@
+module Wasp.Generator.EmailSenders
+  ( getEnabledEmailProvidersJson,
+  )
+where
+
+import Data.Aeson (KeyValue ((.=)), object)
+import qualified Data.Aeson as Aeson
+import qualified Wasp.AppSpec.App.EmailSender as AS.App.EmailSender
+
+getEnabledEmailProvidersJson :: AS.App.EmailSender.EmailSender -> Aeson.Value
+getEnabledEmailProvidersJson emailSender =
+  object $
+    makeProviderJson
+      <$> providersKeyAndName
+  where
+    providersKeyAndName =
+      [ ("isSmtpProviderUsed", AS.App.EmailSender.SMTP),
```

Comment:
Why do you use `used` here when you used `enabled` in other places? Btw enabled sounds better to me -> we know if they are enabled, not if they are used. Not a biggie though, both terms can work, but would be nice if terminology was synced I think.

Reply:
I agree, it's a better and more precise name 👍